jsonutils/gameBinding.go

```diff
@@ -0,0 +1,53 @@
+package jsonutils
+
+import (
+	"errors"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/gin-gonic/gin/binding"
+	"github.com/joshprzybyszewski/cribbage/model"
+)
+
+var _ binding.Binding = gameBinding{}
+var _ binding.BindingBody = gameBinding{}
+var (
+	GameBinding = gameBinding{}
+)
+
+type gameBinding struct {
+	binding.Binding
+	binding.BindingBody
```

Comment:
🔮 If neither of these are used on the struct, we should just not define them as fields.